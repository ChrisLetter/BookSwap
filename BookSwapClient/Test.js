/* eslint-disable prettier/prettier */
import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

const Test = () => {
    return (
        <View style={styles.body}>
            <View style={styles.sectionContainer}>
                <Text style={styles.sectionTitle}>Hello World!</Text>
            </View>
        </View>
    );
};

const styles = StyleSheet.create({
    body: {
        backgroundColor: '#ffffff',
    },
    sectionContainer: {
        marginTop: 32,
        paddingHorizontal: 24,
    },
    sectionTitle: {
        fontSize: 24,
        fontWeight: '600',
        color: '#000000',
    },
});

export default Test;
