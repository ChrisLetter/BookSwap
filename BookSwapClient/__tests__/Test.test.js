/* eslint-disable prettier/prettier */
import React from 'react';
import { cleanup, render } from '@testing-library/react-native';

import Test from '../Test';

afterEach(cleanup);

describe('Test', () => {
    it('should show hello world', () => {
        const helloWorldText = 'Hello World!';

        const { toJSON, getByText } = render(<Test />);

        const foundHelloWorldText = getByText(helloWorldText);

        expect(foundHelloWorldText.props.children).toEqual(helloWorldText);
        expect(toJSON()).toMatchSnapshot();
    });
});