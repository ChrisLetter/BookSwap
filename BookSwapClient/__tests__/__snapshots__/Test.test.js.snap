// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test should show hello world 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
    }
  }
>
  <View
    style={
      Object {
        "marginTop": 32,
        "paddingHorizontal": 24,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontSize": 24,
          "fontWeight": "600",
        }
      }
    >
      Hello World!
    </Text>
  </View>
</View>
`;
